lib/dispatcher.js

```diff
@@ -36,11 +36,10 @@ module.exports = class Dispatcher {
 	 */
 	constructor(request) {
 		this._validateRequest(request);
-
 		this.endpoint = request.endpoint.replace(/^\/?(api\/)?/i, '');
 		this.method = (request.method || 'get').toLowerCase();
-		this.data = request.data || {};
-		this.pristineData = request.data && { ...request.data };
+		this.data = trimObjectValues(request.data && cloneObj(request.data));
+		this.pristineData = request.data && cloneObj(request.data);
```

Comment:
ahora que `this.data` tiene un `cloneObj`, me suena que `this.pristineData` no necesita el clone, si el objeto es muy grande estamos metiendo un segundo clone que no necesitamos.